charts/newrelic-infrastructure-v3/templates/_helpers_compatibility.tpl

```diff
@@ -86,29 +86,45 @@ Returns legacy integrations_config configmap data
 
 {{- define "newrelic.compatibility.message.logFile" -}}
 The `logFile` option is no longer supported and has been replaced by common.agentConfig.log_file.
+
+------
 {{- end -}}
 
 {{- define "newrelic.compatibility.message.resources" -}}
 You have specified the legacy `resources` option in your values, which is not fully compatible with the v3 version.
 This version deploys three different components and therefore you'll need to specify resources for each of them.
-Please use `ksm.resources`, `controlPlane.resources` and `kubelet.resources`.
+Please use
+ - `ksm.resources`,
+ - `controlPlane.resources`
+ - `kubelet.resources`.
```

Comment:
These are commands or subshells. Terminals do not render Markdown... YET.

Weird things could happen in case somebody pipe/script this and do thing as "saving the output of `helm upgrade --yadayada` as an environment variable to print it or something like that.

A inexperienced user could see it also as a copy and paste command like the ones on `info command`. Here you have an example with `info diff`:
![Screenshot 2022-02-16 at 10 27 08](https://user-images.githubusercontent.com/53659978/154234955-558c3fb8-f1e5-4bb7-86a7-617328528362.png)

They put a ` with a ' to avoid copy and pasting or escaping problems.

Reply:
Thanks! I'll replace those